src/App.jsx

```diff
@@ -1,104 +1,19 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+const posts = postsFromServer.map(post => ({
+  user: usersFromServer.filter(el => el.id === post.userId),
```

Comment:
you want to find the user instead of filter the array, so better to call find method